### ISSUE TITLE
[antsibull-meta] Create antsibull Matrix room

### ISSUE THREAD

**Issue by gotmax23** [ansible-community/antsibull-core#146]
@felixfontein, what do you think about creating a Matrix channel `#antsibull:ansible.com` for the antsibull tools? It would be nice to have a dedicated channel for development discussion and for user questions about antsibull-docs, antsibull-changelog, and antsibull-docs-parser (the user-facing antsibull projects) instead of pinging each other in other community rooms.

@GregSutcliffe, is this something you'd be able to help us with if we agree it is a good idea?

**Comment by felixfontein** [comment]
Sounds good to me!

**Comment by GregSutcliffe** [comment]
> @GregSutcliffe, is this something you'd be able to help us with if we agree it is a good idea?

Certainly! In fact, you don't really need me at all, except for one small step :)

Anyone can create a Matrix room and run it as they desire, so go ahead whenever you wish. You can obviously also control the README here, so you can advertise the room, and so on. The only you need our help with is to make it "official" - put it in the Ansible space, add a .com alias, and so on. Let me know once you're ready for that.

**Comment by gotmax23** [comment]
I created https://matrix.to/#/#antsibull:matrix.org. I invited @felixfontein as a room admin. @GregSutcliffe, what are the next steps to make this an official room? Don't I need to invite some Ansible Admin user?

**Comment by GregSutcliffe** [comment]
I've joined the room - if you could promote me to admin I can do the rest and then I'll drop privileges again. Thanks!

**Comment by felixfontein** [comment]
I created PRs to add/update badges to the new room. Right now they are blocked by the badge saying "room not world readable or is invalid".

**Comment by felixfontein** [comment]
The solution above was to enable public history for the room. That's now done. Also, all PRs are merged, so I'm going to close this.
